doc/network.tex

```diff
@@ -135,107 +138,105 @@
 \section*{Version history}
 
 \begin{description}
-\item[Version 0.1, Dec 20, 2018  Draft of the table of contents.]
+\item[Version 0.1, Dec 20, 2018 Draft of the table of contents.]
 \item[Version 0.2, Jan 08, 2019 Structure and outline.]
-\item[Version 0.3, Jan 25, 2019 Start elaborating contents.]
-  Description of state machines; Binary representation.
-\item[Version 0.4, Jan 28, 2019 Extend state machine description]
-\item[Version 0.5, Feb 1,  2019  add PingPong and Block-Fetch protocol]
-\item[Version 0.6, Feb 18, 2019  Overview sections and gluing parts]
-\item[Version 0.7, Feb 25, 2019 ]
-  complete ChainSync Protocol;Request Response Protocol;
-\item[Version 0.8, Mar 11, 2019 ]
-  complete Transaction Submission Protocol
+\item[Version 0.9, Apr 03, 2019 Clean-up todos and WIP remarks]
 \end{description}
 
 \chapter{Overview}
-Ouroboros is a family of new protocols for a proof of stake crypto currency.
+The Cardano blockchain system is based on the Ouroboros family of protocols
+for a proof-of-stake cryptocurrency.
```

Comment:
The operation of these protocols on a node, working in collaboration with the consensus and ledger aspects of Cardano, create the overall distributed system. It is that distributed system that defines the "single source of truth" that is the distributed ledger.

Reply:
I'll take that addition for the next version.
Also this is not the review I promised you with your edits.. this is just the edits from Peter.